### DISCUSSION TITLE
`padding-top` of table does not work. Why?

### DISCUSSION THREAD

**Discussion by wuarmin** [tailwindlabs/tailwindcss#8721] [Help] [answered]
Hey,
with plain CSS a padding can be applied to a `table`-tag. See:
https://jsfiddle.net/jsonimo/6rxb5m2t/1/

but with tailwindcss the padding seems to be ignored? Why is this?
https://play.tailwindcss.com/mljziMjplJ

Thanks

**Comment by davidpollet** [comment, marked as answer]
Hello,
In CSS, you need a border-collapse with a "separate" value to add a padding on table element.
Tailwind comes with a "collapse" value by default.

Add border-separate to your class.

**Reply by wuarmin** [reply]
Thanks! What's the reason for the default border-collapse?

**Reply by davidpollet** [reply]
For Tailwind ? Most tables are collapsed in modern UI. 
It's better to provide the most requested value by default, I guess.
